docs/pipeline.md

```diff
@@ -98,11 +98,12 @@ during component development.
 
 The local runner will try to check if the `base_path` of the pipeline is a local or remote storage. If it's local, the `base_path` will be mounted as a bind volume on every service/component.
 
-If you want to use remote paths (GCS, S3, etc.) you can use the `--extra_volumes` argument to mount extra credentials. This volume will be mounted to every component/service of the docker-compose spec.
+If you want to use remote paths (GCS, S3, etc.) you can use the `--auth-gcp`, `--auth-aws` or `--auth-azure`.
+This will mount your default local cloud credentials to the pipeline. You can also use the `--extra_volumes` argument to mount extra credentials or additional files.
```

Comment:
Maybe we need to make it more explicit that users should make sure to authenticate correctly locally before?

Reply:
Good call, added it